source/install/install-kubernetes-mattermost.rst

```diff
@@ -61,7 +78,98 @@ Save the file as ``mattermost-installation.yaml``:
+Save the file as ``mattermost-installation.yaml``.
+
+**3. Apply the Installation Manifest File**
+
+To initiate deployment, apply the file, specifying the correct path, using:
+
+.. code-block:: sh
+
+  $ kubectl create ns mattermost
+  $ kubectl apply -n mattermost -f /path/to/mattermost-installation.yaml
+
+The deployment process can be monitored in the Kubernetes user interface.
+
+**4. Configure DNS and Use Mattermost**
+
+When the deployment is complete, obtain the hostname or IP address of your Mattermost deployment using the following command:
+
+.. code-block:: sh
+
+  $ kubectl -n mattermost get ingress
+
+Copy the resulting hostname or IP address from the ``ADDRESS`` column, open your browser, and connect to Mattermost.
+
+Use your domain registration service to create a canonical name or IP address record for the ``ingressName`` in your manifest,
+pointing to the address you just copied. For example, on AWS you would do this within a hosted zone in Route53.
+
+Navigate to the ``ingressName`` URL in your browser and use Mattermost.
+
+Restoring an Existing Mattermost MySQL Database
+-----------------------------------------------
+
+You can leverage the Mattermost Operator to utilize an existing Mattermost MySQL database with a new Mattermost installation, in its own namespace.
+The steps you follow to create and upload your backup depends on the provider you're using and your use case. It's
+recommended that you consult the relevant documentation or, if your deployment is managed in a different way, consult your Administrator.
+
+It is important to note that this process requires the creation of a new Mattermost
+installation - editing the existing ``.yaml`` files is not recommended and can result in data loss.
+
+The process described below needs to be completed prior to proceeding with the Mattermost deployment.
+
+1. Create a backup of your database (e.g., using *mysqldump*).
+2. Deploy a new server (e.g., an AWS instance).
+3. Install a backup program and back up the database on the new server/instance.
+4. Upload the backed up database to your cloud storage provider (e.g., Amazon S3).
+5. Create a ``secret.yaml``` file:
+
+Open a text editor and create a text file containing your credentials which will be used to access the uploaded database.
+
+Save the file as ``secret.yaml``. The example below is for AWS/S3.
+
+.. code-block:: yaml
+
+   apiVersion: v1
+   kind: Secret
+   metadata:
+    name: test-restore
+   type: Opaque
+   stringData:
+    AWS_ACCESS_KEY_ID: XXXXXXXXXXXX
+    AWS_SECRET_ACCESS_KEY: XXXXXXXXXXXX/XXXXXXXXXXXX
+    AWS_REGION: us-east-1
+    S3_PROVIDER: AWS
+
+**Parameters**
+- ``name``. The name of this manifest which is referenced in the installation manifest.
+
+
+6. Create a Mattermost cluster installation manifest.
+
+Open a text editor and create a text file with the following details. Save the file as ``mattermost-installation.yaml``:
+
+.. code-block:: yaml
+
+  apiVersion: mattermost.com/v1alpha1
```

Comment:
How will customers know when the API version is updated and is there an easy way for them to change this?

Reply:
This is a common/best practice pattern in k8s. When the api version changes, they can start using it by just applying a new manifest that use the new version

Reply:
Could they update the existing manifest with the new API version? Or must a new one be created?

Reply:
To follow on from that, should we include a link to the repo and say something like "Check the repo for details of API version updates"?